src/components/organize/tasks/TaskActionButtons/index.tsx

```diff
@@ -56,46 +64,39 @@ const TaskActionButtons: React.FunctionComponent<TaskActionButtonsProps> = ({ ta
                 <PublishButton task={ task }/>
             </Box>
             <Box>
-                <Button
-                    color="secondary"
-                    data-testid="task-action-buttons-menu-activator"
-                    disableElevation
-                    onClick={ (e) => setMenuActivator(e.currentTarget) }
-                    variant="contained">
-                    <Settings />
-                </Button>
+                <ZetkinEllipsisMenu items={ [
```

Comment:
Same question here re `TabbedLayout`.

Reply:
Yes I understand. Thinking about it, I think it's probably not a great idea, because these menu items have dependencies such as: mutations, states for handling dialogs and dialogs themselves. This adds lots of code to the layout that would be nicely encapsulated in `ActionButtons` components. I did just start moving them over to see how it looks and tbh I don't love how it makes the layout handle everything.

Reply:
My mind has been changed.